backend/src/middleware/hashtags/hashtagsMiddleware.spec.js

```diff
@@ -129,10 +148,29 @@ describe('hashtags', () => {
         )
       })
 
-      describe('afterwards update the Post by removing a Hashtag, leaving a Hashtag and add a Hashtag', () => {
-        // The already existing Hashtag has no class at this point.
-        const postContent =
-          '<p>Hey Dude, <a class="hashtag" href="/search/hashtag/Elections">#Elections</a> should work equal for everybody!? That seems to be the only way to have equal <a href="/search/hashtag/Liberty">#Liberty</a> for everyone.</p>'
+      describe('updates the Post by removing, keeping and adding one hashtag respectively', () => {
```

Comment:
This is a false successful test !!! 🤔
Because the `data-hashtag-id` is deleted by saving the content in the database.
On reedit it is no link anymore.
Therefore here you create a test that’s not existent in reality.
That’s why we used the `href` to search for a hashtag.